consent-ws/src/main/java/org/broadinstitute/consent/http/models/DataUseDTO.java

```diff
@@ -272,5 +281,20 @@ public void setNonBiomedical(Boolean nonBiomedical) {
     public String toString() {
         return new GsonBuilder().create().toJson(this);
     }
-    
+
+    public static DataUseDTO parseDataUse(String str) throws IOException {
+        if (str == null || str.isEmpty()) {
```

Comment:
Would using `java.util.Optional` alleviate the `null` pain?

Reply:
I would really like for it to be helpful, but I think it just adds extra code.
```
        Optional<String> opt = Optional.ofNullable(str);
        if (!opt.isPresent() || opt.get().isEmpty()) {
            return null;
            ...
```

Reply:
The way to use `Optional` would be to include it in the type signature 

```java
public static DataUseDTO parseDataUse(String str) { ... }
```

like we do in Scala so `parseDataUse` wouldn't throw and any errors would be handled upstream of this method.

I can't assess if this is worthwhile, though.

Reply:
Actually, returning an optional does help us with the mapper since we can handle exceptions much better!

Reply:
Yes, actually, the Optional approach is much cleaner, thanks!